charactersheet/charactersheet/index.html

```diff
@@ -73,100 +73,131 @@ <h2 data-bind='text: playerTitle'></h2>
-				<div class="col-md-4 col-sm-6 col-padded">
-					<div data-bind="template: { name: 'feat_traits.tmpl', templateUrl: 'feat_traits', data: featuresTraitsViewModel }">
-					</div>
+			</div>
+		</div>
+		<div class="row">
+			<div class="col-md-4 col-sm-12 col-padded">
+				<div data-bind="template: { name: 'ability.tmpl', templateUrl: 'ability_scores', data: abilityScoresViewModel }">
 				</div>
 			</div>
-			<img src="/images/heart.png" height="35" style="margin-top:-12px;" />&nbsp;
-			<h3 style="display:inline;">
-				Health &amp; Stats
-			</h3>
-      <hr />
-			<div class="row">
-				<div class="col-md-8">
-					<div id="stats" data-bind="template: { name: 'stats.tmpl', templateUrl: 'stats', data: statsViewModel }">
-					</div>
+			<div class="col-md-4 col-sm-12 col-padded">
+				<div id="spell_stats" data-bind="template: { name: 'spell_stats.tmpl', templateUrl: 'spell_stats', data: spellStatsViewModel }">
 				</div>
-				<div class="col-md-4">
-					<div id="spell_stats" data-bind="template: { name: 'spell_stats.tmpl', templateUrl: 'spell_stats', data: spellStatsViewModel }">
-					</div>
+			</div>
+			<div class="col-md-4 col-sm-12 col-padded">
+				<div id="saving_throws" data-bind="template: { name: 'saving_throws.tmpl', templateUrl: 'saving_throws', data: savingThrowsViewModel }">
 				</div>
 			</div>
-			<img src="/images/shield.png" height="35" style="margin-top:-12px;" />&nbsp;
-			<h3 style="display:inline;">
-				Equipment, Skills, &amp; Treasure
-			</h3>
-      <hr />
-			<div class="row">
-				<div class="col-md-4 col-sm-6 col-padded">
-					<div id="spell_slots" data-bind="template: { name: 'spell_slots.tmpl', templateUrl: 'spell_slots', data: spellSlotsViewModel }">
-					</div>
-					<div id="saving_throws" data-bind="template: { name: 'saving_throws.tmpl', templateUrl: 'saving_throws', data: savingThrowsViewModel }">
-					</div>
-					<div id="skills" data-bind="template: { name: 'skills.tmpl', templateUrl: 'skills', data: skillsViewModel }">
-					</div>
+		</div>
+	</div>
+<!-- End Stats Tab -->
+	<div role="tabpanel" class="tab-pane" id="skills" data-bind="with: skillsTabViewModel, css: skillsTabStatus">
+		<div class="row">
+			<div class="col-md-6 col-sm-12 col-padded">
+				<div data-bind="template: { name: 'feats_prof.tmpl', templateUrl: 'feats_prof', data: featsProfViewModel }">
 				</div>
-				<div class="col-md-8 col-sm-6 col-padded">
-					<div id="equipment" data-bind="template: { name: 'equipment.tmpl', templateUrl: 'equipment', data: equipmentViewModel }">
-					</div>
-					<div id="spells" data-bind="template: { name: 'spells.tmpl', templateUrl: 'spells', data: spellbookViewModel }">
-					</div>
-					<div data-bind="template: { name: 'treasure.tmpl', templateUrl: 'treasure', data: treasureViewModel }">
-					</div>
+			</div>
+			<div class="col-md-6 col-sm-12 col-padded">
+				<div id="skills" data-bind="template: { name: 'skills.tmpl', templateUrl: 'skills', data: skillsViewModel }">
```

Comment:
Looks like a similar issue with the skills.

![screenshot 2016-02-03 09 58 26](https://cloud.githubusercontent.com/assets/3310280/12791729/4d91865e-ca5d-11e5-9cda-f3dc6dbad2d7.png)